src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -945,4 +945,11 @@ Do you want to proceed?</value>
   <data name="Error_creating_instance_of_CodeFixProvider" xml:space="preserve">
     <value>Error creating instance of CodeFixProvider</value>
   </data>
+  <data name="Adding_missing_import_statements" xml:space="preserve">
+    <value>Adding missing import statements...</value>
+    <comment>Used as a thread awaited dialog letting the user know that work is being done on paste</comment>
+  </data>
+  <data name="Add_missing_imports_on_paste" xml:space="preserve">
+    <value>Add Missing Imports On Paste</value>
```

Comment:
Should the string name reflect the capitalization?

Reply:
I never know, but  sure :)

Reply:
where is this shown?  the capitalization seems super weird.    Also, even if it's Title-Case, then `on` would not be capitalized.